src/sentry/lang/native/symbolicator.py

```diff
@@ -59,13 +59,14 @@
         "appconnectKey": {"type": "string", "minLength": 2, "maxLength": 20},
         "appconnectPrivateKey": {"type": "string"},
         "itunesUser": {"type": "string", "minLength": 1, "maxLength": 100},
-        "itunesCreated": {"type": "string"},
+        "itunesCreated": {"type": "string", "format": "date-time"},
```

Comment:
I don't always get the `type`, `name` and `id` from the response, so we will have to remove them from the required list or update the endpoints to always provide them

Reply:
hmm, i should fix it so you always get those instead.  they can never be missing as too many other things depend on this